apps/elixir_chatbot_core/lib/embedding_model/sentence_transformers.ex

```diff
@@ -0,0 +1,55 @@
+defmodule ElixirChatbotCore.EmbeddingModel.SentenceTransformers do
+  alias ElixirChatbotCore.EmbeddingModel.SentenceTransformers
+  defstruct [:model_name]
+
+  @spec start_semantics_server :: :ignore | {:error, any} | {:ok, pid}
+  def start_semantics_server() do
+    Semantics.start_link()
+  end
+
+  @spec start_semantics_server(String.t()) :: :ignore | {:error, any} | {:ok, pid}
+  def start_semantics_server(model_name) do
+    Semantics.start_link(model_name)
+  end
+
+  @spec load_model(String.t()) :: any
+  def load_model(model_name) do
+    Semantics.load(model_name)
+  end
+
+  @spec new(String.t()) :: %ElixirChatbotCore.EmbeddingModel.SentenceTransformers{model_name: String.t()}
```

Comment:
Alias for `ElixirChatbotCore.EmbeddingModel.SentenceTransformers` is already declared, so I think it would be cleaner to use it here, too